CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### Upcoming Changes
 * 0.11 Support
+    * Add support for hints `PACK_MODN_DIV_MODN` and `XS_MOD` [#991](https://github.com/lambdaclass/cairo-rs/pull/991)
```

Comment:
It's probably a good idea (and consistent with previous entries) to include the Python code in a code block for users to know what they would need to write to use the whitelisted hint.